web/src/apps/selector.ts

```diff
@@ -0,0 +1,179 @@
+import { apply, css, tw } from '@twind/core'
```

Comment:
I think this file should go into a new `ui` (or similar name) folder next to `apps` given that the web components in `nodes` are likely to use these UI components too.

Reply:
ha! I've done this with the new work I'm looking at today! Will migrate that into this PR. Bear with me a moment!